outputs.tf

```diff
@@ -6,3 +6,8 @@ output "usage_IAM_roles" {
     "roles/pubsub.viewer",
   ]
 }
+
+output "topic_id" {
+  description = "an identifier of the topic with format projects/{{project}}/topics/{{name}}"
```

Comment:
Again inconsistent description start- use capital like you did for others. 🥇

Reply:
Crap